doc/options.yaml

```diff
@@ -491,11 +491,14 @@ oversetUpdateMode:
     This is the cheapest option to use when the entire overset mesh is warped together, such as with USMesh in IDWarp.
   fast: >
     Only the weights are updated and the donors are unchanged.
-    This is usually a fairly small correction and is fast to run.
     This is used when the entire overset mesh is warped together, such as with USMesh in IDWarp.
+    This is fast to run but only works for small design changes.
+    Large design changes will cause the solver to stall because of nonphysical interpolation weights.
```

Comment:
What is small and what is large? I am guessing a "large" design change is where the cell centers are displaced enough that the cell center is outside the dual mesh of the cell centers from the parent cells? If so, can't we throw an error when someone wants the fast mode but some weights gets less than 0 or greater than 1?

Reply:
This is correct. I added the check.